onnx/defs/gen_doc.py

```diff
@@ -46,20 +46,22 @@ def main(args):
           "            Do not modify directly and instead edit operator definitions.*\n")
 
       sorted_ops = sorted(
-          (int(schema.support_level), op_type, schema)
+          (schema.domain, int(schema.support_level), op_type, schema)
           for (op_type, schema) in defs.get_all_schemas().items())
 
       fout.write('\n')
 
       # Table of contents
-      for _, op_type, schema in sorted_ops:
-          s = '* <a href="#{}">{}{}</a>\n'.format(
-              op_type, support_level_str(schema.support_level), op_type)
+      for domain, _, op_type, schema in sorted_ops:
+          if domain:
+              domain = ' (Domain: {})'.format(domain)
+          s = '* <a href="#{}">{}{}</a>{}\n'.format(
+              op_type, support_level_str(schema.support_level), op_type, domain)
           fout.write(s)
 
       fout.write('\n')
 
-      for _, op_type, schema in sorted_ops:
+      for domain, _, op_type, schema in sorted_ops:
```

Comment:
nit: since you don't use `domain` in this block, you can do `for _, _, op_type, schema ...`